charts/karpor/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 name: karpor
-version: 0.7.5
+version: 0.7.6
 type: application
-appVersion: 0.6.3
+appVersion: 0.6.3-alpha.2
 description: A modern kubernetes visualization tool (Karpor).
```

Comment:
Using an alpha version (0.6.3-alpha.2) for appVersion indicates this is a pre-release version. Ensure users are aware this is not a stable release.